Source/Android/jni/MainAndroid.cpp

```diff
@@ -604,6 +606,31 @@ JNIEXPORT void JNICALL Java_org_dolphinemu_dolphinemu_NativeLibrary_ReloadConfig
   SConfig::GetInstance().LoadSettings();
 }
 
+JNIEXPORT void JNICALL Java_org_dolphinemu_dolphinemu_NativeLibrary_SetOverlaySIDevice(
+    JNIEnv* env, jobject obj, jint gcController)
+{
+  SerialInterface::SIDevices SIDevice;
+
+  SConfig::GetInstance().LoadSIDevice0Setting();
+
+  switch (gcController)
```

Comment:
You can skip this entire switch statement with `si_device = static_cast<SerialInterface::SIDevices>(gcController)`.